### PR TITLE
Adds notes for 4.6.61 release

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for 4.6.61 release. Links will not work yet.

Link to docs preview:
http://file.rdu.redhat.com/opayne/4.6.61/release_notes/ocp-4-6-release-notes.html#ocp-4-6-61

### CODE REVIEW

**Review by Wiharris — COMMENTED** [review]
/LGTM

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -3790,3 +3790,22 @@ $ oc adm release info 4.6.60 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.6.61 --pullspecs
+----
+
+[id="ocp-4-6-61-updating"]
+==== Updating
+
+To update an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-6-release-notes.adoc:3811]: LGTM

**Review by JoeAldinger — COMMENTED** [review]
LGTM

**Pull request merged**